CRM/Core/Payment/BaseIPN.php

```diff
@@ -224,22 +224,25 @@ public function pending(&$objects, &$transaction) {
   /**
    * Process cancelled payment outcome.
    *
+   * @deprecated The intended replacement code is
+   *
+   * Contribution::update(FALSE)->setValues([
```

Comment:
@eileenmcnaughton It's fairly obvious to me but can you use the full class here ie. `\Civi\API4\Contribution::..` just to make it obvious we're using API4.

Reply:
@mattwire I think our preferred approach is the name-spacing approach - ie  declare the namespace in a use statement & then use the short name. We've done that in quite a lot of places. As it becomes really prevalent to use the v4 api people will start to assume it anyway.